invenio_administration/assets/semantic-ui/js/invenio_administration/src/actions/Edit.js

```diff
@@ -1,16 +1,47 @@
+import PropTypes from "prop-types";
 import React, { Component } from "react";
 import { i18next } from "@translations/invenio_administration/i18next";
-import { Button } from "semantic-ui-react";
+import { Button, Popup, Icon } from "semantic-ui-react";
+import Overridable from "react-overridable";
 
-export class Edit extends Component {
+class EditCmp extends Component {
   render() {
+    const { display, editUrl, disable, disabledMessage, resource } = this.props;
+    if (!display) {
+      return null;
+    }
+    const disabled = disable(resource);
+
     return (
-      <Button
-        icon="edit"
-        labelPosition="left"
-        content={i18next.t("Edit")}
-        {...this.props}
+      <Popup
+        content={disabledMessage}
+        disabled={!disabled}
+        trigger={
+          <span className="mr-5">
+            <Button as="a" disabled={disabled} href={editUrl} icon labelPosition="left">
```

Comment:
```suggestion
            <Button as="a" disabled={disabled} href={editUrl} icon labelPosition="left" className="mr-5" title={disabledMessage}>
```
margin: If the span can be removed

disabledMessage: I don't think the popup content will be read to the screen reader, so the `title` is just to make sure it's announced when focusing the button with a screen reader. (If `disabledMessage` is defined even when the button is not disabled, it'll need a conditional - but I think it is null in that case?)

Reply:
it is not null, it is coming from default props, so it will be present at all times. In addition I find it confusing to put disabled message on the button, since it should be the popup's responsibility to inform about this. Is there any other way to make the popup readable by screen reader? @jennur

Reply:
So apparently this is the equivalent to `title`: `<Button aria-describedby="popup-id" />` and `<Popup id="popup-id" role="tooltip />` [w3.org](https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/). But that only works when the tooltip (popup) is available in the DOM, and since that is not the case for `<Popup />`, the tooltip will never be announced by the screenreader by doing it that way.